packages/image_picker/image_picker_ios/test/image_picker_ios_test.dart

```diff
@@ -878,6 +895,174 @@ void main() {
     });
   });
 
+  group('#getMedia', () {
+    test('calls the method correctly', () async {
+      log.returnValue = <String>['0', '1'];
+      await picker.getMedia(options: MediaSelectionOptions());
+
+      expect(
+        log.calls,
+        <_LoggedMethodCall>[
+          const _LoggedMethodCall('pickMedia', arguments: <String, dynamic>{
+            'maxWidth': null,
+            'maxHeight': null,
+            'imageQuality': null,
+            'requestFullMetadata': true,
+            'allowMultiple': true
```

Comment:
Separate tests for each of these bool arguments on their own would be useful. Right now, if someone made the mistake that is trivial to make and swapped the bools, it looks like the tests would still pass.

Reply:
Where are these tests?